ckanext/showcase/controller.py

```diff
@@ -54,6 +54,9 @@ def new(self, data=None, errors=None, error_summary=None):
         except NotAuthorized:
             abort(401, _('Unauthorized to create a package'))
 
+        # get discuss name
+        c.disqus_name = config.get('disqus.name')
```

Comment:
Using `c` to pass variables is frown upon in modern CKAN versions. Can you instead pass the `disqus_name` explicitly as an `extra_var` to the template?

Reply:
Ah, I now see that we are not rendering the template directly in this class which makes this difficult. Can you create a `h.disqus_name()` template helper instead that returns the config value if present? I know it's a pain but it's more robust this way across CKAN versions.